app/uk/gov/hmrc/pillar2/service/RegistrationService.scala

```diff
@@ -70,19 +68,32 @@ class RegistrationService @Inject() (
       true
     )
   }.getOrElse {
-    logger.warn(s"[Session ID: ${SessionIdHelper.sessionId(hc)}] - RegistrationService - Filing Member Registration Information Missing")
+    logger.warn("Filing member registration failed as one or more required fields were missing")
     registerWithoutIdError
   }
 
+  def registerNewFilingMember(userAnswers: UserAnswers)(implicit hc: HeaderCarrier) =
```

Comment:
Add return type annotation.

Reply:
Done

Reply:
rename method to registerNoIdNewFilingMember to distinguish between NoId and Id journeys for new filing member.